go-controller/pkg/clustermanager/pod/allocator.go

```diff
@@ -100,8 +101,8 @@ func (a *PodAllocator) Init() error {
 
 // getActiveNetworkForNamespace returns the active network for the given namespace
 // and is a wrapper around util.GetActiveNetworkForNamespace
-func (a *PodAllocator) getActiveNetworkForNamespace(namespace string) (string, error) {
-	return util.GetActiveNetworkForNamespace(namespace, a.nadLister)
+func (a *PodAllocator) findActiveNetworkForNamespace(namespace string) (*ovncnitypes.NetConf, error) {
```

Comment:
why is it `find` and not `get`?

Reply:
The `get` is usually use for stuff like field or similar, while  `find` is more related to databases lookup.  We are looking at the etcd databse to find the nads here, so it make more sense to use `find`

Also function users tend to be more careful on using something like `find` it "feels" more costly.